### ISSUE TITLE
PHP inside <script> usually breaks syntax highlighting.

### ISSUE THREAD

**Issue by octoxan** [atom/language-php#165]
Inside of a php file if I do this...

```php
<script>
    var foo = <?php echo json_encode('bar'); ?>
</script>
```

![broken](https://cloud.githubusercontent.com/assets/4382195/19906267/746ab754-a050-11e6-8f90-ef5232a87458.png)

syntax highlighting is broken for the entire rest of the page.

Is there a way around this? I see that it does it for Githubs syntax highlighting as well. The closing <script> and any HTML/JS/PHP after it are all colored incorrectly.

**Comment by 50Wliu** [comment]
This has been fixed and should be available starting in Atom 1.13.0.

**Comment by octoxan** [comment]
Thank goodness this had been driving me crazy.
